<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Task</class>
 <widget class="QDialog" name="Task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>add/modifyTask</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/widgets_icons/document.png</normaloff>:/icons/widgets_icons/document.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <spacer name="horizontalSpacer_14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPlainTextEdit" name="plainTextEdit_Name">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ToDo_2">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>具体内容</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QPlainTextEdit" name="plainTextEdit_Details">
         <property name="maximumSize">
          <size>
           <width>100000</width>
           <height>300</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="plainText">
          <string>请输入具体的细节</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ToDo">
         <property name="font">
          <font>
           <family>仿宋</family>
           <pointsize>13</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>计划名称</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_type">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>事件类型</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButton_Once">
            <property name="font">
             <font>
              <family>仿宋</family>
              <pointsize>11</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>单次</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_Repeat">
            <property name="text">
             <string>重复型</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_Deadline">
            <property name="text">
             <string>日期截止型</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_priority">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>事件优先级</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButton_high">
            <property name="text">
             <string>高</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/widgets_icons/icons8-red-circle-48.png</normaloff>:/icons/widgets_icons/icons8-red-circle-48.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_medium">
            <property name="text">
             <string>中</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/widgets_icons/icons8-yellow-circle-48.png</normaloff>:/icons/widgets_icons/icons8-yellow-circle-48.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_low">
            <property name="text">
             <string>低</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icons/widgets_icons/icons8-green-67.png</normaloff>:/icons/widgets_icons/icons8-green-67.png</iconset>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget_types">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_Once">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>56</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_start">
           <property name="font">
            <font>
             <family>仿宋</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>开始时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_start_once"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>57</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_end">
           <property name="font">
            <font>
             <family>仿宋</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>结束时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_end_once">
           <property name="displayFormat">
            <string>yyyy/MM/dd HH:mm</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>56</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Repeat">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_startDate">
             <property name="font">
              <font>
               <family>仿宋</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>开始日期</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_start_repeat"/>
           </item>
           <item>
            <widget class="QLabel" name="label_endDate">
             <property name="font">
              <font>
               <family>仿宋</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>结束日期</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_end_repeat"/>
           </item>
           <item>
            <widget class="QLabel" name="label_start_5">
             <property name="font">
              <font>
               <family>仿宋</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>的</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timeEdit_start_repeat"/>
           </item>
           <item>
            <widget class="QLabel" name="label_start_6">
             <property name="font">
              <font>
               <family>仿宋</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>到</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTimeEdit" name="timeEdit_end_repeat"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkBox_everyDay">
              <property name="text">
               <string>每天</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkBox_Monday">
              <property name="text">
               <string>每周一</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="checkBox_TuesDay">
              <property name="text">
               <string>每周二</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="checkBox_Wednesday">
              <property name="text">
               <string>每周三</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="checkBox_Thursday">
              <property name="text">
               <string>每周四</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="checkBox_Friday">
              <property name="text">
               <string>每周五</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="checkBox_Saturday">
              <property name="text">
               <string>每周六</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QCheckBox" name="checkBox_Sunday">
              <property name="text">
               <string>每周日</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_Deadline">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>161</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_end_deadline">
           <property name="font">
            <font>
             <family>仿宋</family>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>截止时间</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit_deadline"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>160</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_accept">
         <property name="text">
          <string>确定</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/widgets_icons/checkmark-verify-interface-symbol-button.png</normaloff>:/icons/widgets_icons/checkmark-verify-interface-symbol-button.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="text">
          <string>取消</string>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/icons/widgets_icons/close-circular-button-symbol.png</normaloff>:/icons/widgets_icons/close-circular-button-symbol.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
